modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/MyWorkflowTasks.testcase

```diff
@@ -315,12 +315,45 @@ definition {
 	}
 
 	@description = "Verify that the user can order the results inside Assigned to Me by Due Date"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanOrderAssignedtoMeTasksByDueDate {
+		ApplicationsMenu.gotoPortlet(
+			category = "Workflow",
+			panel = "Applications",
+			portlet = "Process Builder");
+
+		Navigator.gotoNavItem(navItem = "Configuration");
+
+		Workflow.configureWorkflow(
+			workflowDefinition = "Single Approver",
+			workflowResourceValue = "Blogs Entry");
+
+		JSONBlog.addEntry(
+			entryContent = "Blogs Entry Content",
+			entryTitle = "Blogs Entry");
+
+		UserBar.gotoDropdownItem(dropdownItem = "My Workflow Tasks");
+
+		Navigator.gotoNavTab(navTab = "Assigned to My Roles");
+
+		Workflow.updateDueDateTaskByActions(
+			workflowAssetTitle = "Blogs Entry",
+			workflowAssetType = "Blogs Entry",
+			workflowDueDateTime = "1:45 PM",
+			workflowTask = "Review");
```

Comment:
In this piece, you can create and them update the due date of Blogs Entry 2 to a date closer, and Blogs Entry 1 to a later date, then, assign both to you and activate the Due Date filter, use a macro to ensure the expected position.